packages/core/core/src/Validation.js

```diff
@@ -90,8 +90,9 @@ export default class Validation {
               await Promise.all(
                 assets.map(async asset => {
                   let config = null;
-                  if (plugin.getConfig) {
-                    config = await plugin.getConfig({
+                  let getConfig = plugin.getConfig?.bind(plugin);
+                  if (getConfig) {
+                    config = await getConfig({
```

Comment:
This can also be worked around by factoring `let publicAsset = new Asset()` out before the if statement